Bottom: 39a9628cfe6781707faef85cc68ba8d496c51770
Top:    34177299a84ed28447de014b11aca8c0ffc6a929
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 15:20:05 +0000

w32: TMP Build fixes


---

diff --git a/w32/build-all.sh b/w32/build-all.sh
index 04aec82..f6a641c 100755
--- a/w32/build-all.sh
+++ b/w32/build-all.sh
@@ -2,9 +2,6 @@ NEVERBUILD="
 "
 
 NOBUILD="
-"
-
-BUILD="
 jpeg
 zlib
 libpng
@@ -12,6 +9,9 @@ tiff
 freetype
 pixman
 libiconv
+"
+
+BUILD="
 gettext
 glib
 atk
